homeassistant/components/air_quality/__init__.py

```diff
@@ -50,10 +50,12 @@
     "sulphur_dioxide": ATTR_SO2,
 }
 
+# mypy: disallow-any-generics
+
```

Comment:
_Not something for this PR._

It might make sense and update `mypy_config` so that we can specify that somewhere else. Including the comment in each integration did work for one or two. At scale, this doesn't seem to be the best choice.